src/main/java/dinkplugin/DinkPlugin.java

```diff
@@ -271,5 +272,20 @@ public void onWidgetLoaded(WidgetLoaded event) {
                 }
             }
         }
+
+        final int SPEEDRUN_COMPLETED_GROUP_ID = 781;
+        final int SPEEDRUN_COMPLETED_QUEST_NAME_CHILD_ID = 4;
+        final int SPEEDRUN_COMPLETED_DURATION_CHILD_ID = 10;
+        final int SPEEDRUN_COMPLETED_PB_CHILD_ID = 12;
+        if (config.notifySpeedrun() && groupId == SPEEDRUN_COMPLETED_GROUP_ID) {
+            Widget questName = client.getWidget(SPEEDRUN_COMPLETED_GROUP_ID, SPEEDRUN_COMPLETED_QUEST_NAME_CHILD_ID);
+            Widget duration = client.getWidget(SPEEDRUN_COMPLETED_GROUP_ID, SPEEDRUN_COMPLETED_DURATION_CHILD_ID);
+            Widget personalBest = client.getWidget(SPEEDRUN_COMPLETED_GROUP_ID, SPEEDRUN_COMPLETED_PB_CHILD_ID);
+            if (questName == null || duration == null || personalBest == null) {
+                log.error("Found speedrun finished widget (group id {}) but it is missing something, questName={}, duration={}, pb={}", SPEEDRUN_COMPLETED_GROUP_ID, questName, duration, personalBest);
+            }
+//            log.info("quest name is {}, duration: {}, pb: {}", questName.getText(), duration.getText(), personalBest.getText());
+            this.speedrunNotifier.attemptNotify(Utils.parseQuestWidget(questName.getText()), duration.getText(), personalBest.getText());
```

Comment:
```suggestion
            this.speedrunNotifier.attemptNotify(Utils.parseQuestWidget(questName.getText()), duration.getText(), personalBest.getText());
            this.levelNotifier.reset();
```

Reply:
What does this do? Skip notifications?

Reply:
Yes, if `LevelNotifier.currentLevels` is not cleared after going into a speedrun world, when the player hops to a normal world, the new levels will be received, which would look like a level up. Reset makes the skill levels null, so no notification occurs (`https://github.com/pajlads/DinkPlugin/blob/c48b91905eb96095fdf446376077b604db33413a/src/main/java/dinkplugin/LevelNotifier.java#L83`)

`levelNotifier.reset()` should also be called if the speedrun is cancelled early (i.e., started but not completed & world hop occurs)

Reply:
Actually maybe skipping notifications should be a separate PR